Dockerfile

```diff
@@ -1,19 +1,18 @@
-ARG PHP_VERSION=8.1
-
-FROM php:${PHP_VERSION}-cli
+FROM php:8.1-cli
 
+# Install extensions
 RUN apt-get update && \
     apt-get install -y autoconf pkg-config libssl-dev git unzip libzip-dev zlib1g-dev && \
     pecl install mongodb && docker-php-ext-enable mongodb && \
     pecl install xdebug && docker-php-ext-enable xdebug && \
     docker-php-ext-install -j$(nproc) zip
 
-COPY --from=composer:2.6.2 /usr/bin/composer /usr/local/bin/composer
-
-ENV COMPOSER_ALLOW_SUPERUSER=1
-
-WORKDIR /code
+# Increase memory limit
+RUN cp /usr/local/etc/php/php.ini-development /usr/local/etc/php/php.ini && \
+    echo "memory_limit=-1" >> /usr/local/etc/php/php.ini
```

Comment:
Memory limit increased.

Reply:
The memory limit in phpunit.xml is enough.
Could you merge the `RUN` stages for php.ini copy and xdebug mode.